Library/ViewModels/PledgeViewModelTests.swift

```diff
@@ -1102,31 +1117,242 @@ final class PledgeViewModelTests: TestCase {
       self.vm.inputs.configureWith(project: .template, reward: .template, refTag: .activity, context: .pledge)
       self.vm.inputs.viewDidLoad()
 
+      self.beginSCAFlowWithClientSecret.assertDidNotEmitValue()
+      self.submitButtonEnabled.assertValues([false])
+      self.goToThanks.assertDidNotEmitValue()
+      self.showErrorBannerWithMessage.assertDidNotEmitValue()
+
+      self.vm.inputs.creditCardSelected(with: "123")
+
+      self.submitButtonEnabled.assertValues([false])
+
+      self.vm.inputs.pledgeAmountViewControllerDidUpdate(
+        with: (amount: 25.0, min: 10.0, max: 10_000.0, isValid: true)
+      )
+
+      self.submitButtonEnabled.assertValues([false, true])
+
+      self.vm.inputs.submitButtonTapped()
+
+      self.beginSCAFlowWithClientSecret.assertDidNotEmitValue()
+      self.submitButtonEnabled.assertValues([false, true, false])
+      self.goToThanks.assertDidNotEmitValue()
+      self.showErrorBannerWithMessage.assertDidNotEmitValue()
+
+      self.scheduler.run()
+
+      self.beginSCAFlowWithClientSecret.assertDidNotEmitValue()
+      self.submitButtonEnabled.assertValues([false, true, false, true])
+      self.goToThanks.assertDidNotEmitValue()
+      self.showErrorBannerWithMessage.assertValues(
+        ["The operation couldn’t be completed. (KsApi.GraphError error 5.)"]
```

Comment:
This is a strange error message. Shouldn't this be "Something went wrong." which is the `localizedDescription` of `GraphError.invalidInput`?